src/SIL/Parser.hs

```diff
@@ -361,10 +375,20 @@ parseTopLevel = do
 
 debugIndent i = show $ State.runState i (initialPos "debug")
 
+initialMap = fromList
+  [ ("zero", TZero)
+  , ("left", TLam (Open (Right "x")) . TLeft . TVar . Right $ "x")
```

Comment:
We can actually used the Closed TLam form, because these functions don't refer to any variables bound in a greater scope. Open works, but Closed should be slightly faster to evaluate.